src/main/java/spotify/api/enums/HttpStatusCode.java

```diff
@@ -1,5 +1,15 @@
 package spotify.api.enums;
 
+/**
+ * This class contains and sets the HTTP Status codes when using POST and GET HTTP requests.
+ * <p>
+ * For further information see: <a href="https://developer.spotify.com/documentation/general/guides/authorization-guide/">Authorization Guide</a>
```

Comment:
This is not only used for authorization so refering to these documentation is not correct.

Reply:
Is there a specific documentation to refer to? I cannot seem to find any info on HTTP status codes